contact.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+
+<html lang="en-GB">
+
+	<head>
+		<title>Contact Page</title>
+		<link rel="stylesheet" type="text/css" href="portfolio_style.css">
+	</head>
+		
+	<body>
+		
+		<nav class="nav_bar">
+			<p>Karen Schulz</p>
+			
+			<div>
+				<a href="portfolio_home.html">Home</a>
+				<a href="portfolio.html">Portfolio</a>
+			</div>
+			
+		</nav>
+		
+		<article>
+			<div class="contact"><img class="picture-of-me" src="images/half_body.png" alt="my_profile_picture">
+		<br>
```

Comment:
shouldn't be using br. You should accomplish the styling using css instead.
so if you wanted more space above the p tag then you could add padding-top to it in css

Reply:
br erased

Reply:
and followed your advise of using div, and styled them in my CSS. Thanks